src/styles/card.scss

```
@@ -0,0 +1,99 @@
+.product {
+  border: 1px solid #f3f3f3;
+  border-radius: $borderRadius;
+  height: $heightProduct;
+  width: $widthProduct;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-top: 13px;
+    padding-top: 14px;
+    box-sizing: content-box;
+    transition: transform 1s ease-in-out;
+    border: 3px solid  transparent;
+
+    &:hover {
+      transform: scale(1.2);
+      border-color: #f3f3f3;
+    }
+  }
+
+  &__card {
+    padding-left: $padding;
+    padding-right: $padding;
+    margin-top: 42px;
+  }
+
+  &__title {
+    @include sizeFont-2;
+  }
+
+  &__code {
+    margin-top: 5px;
+    @include sizeFont;
+  }
+
+  &__rating {
+    @include spaceBetween;
+    margin-top: 17px;
+  }
+
+  &__rating-stars {
+    display: flex;
+  }
+
+  &__rating-star {
+    background-image: url("../images/star-active.svg");
+    background-repeat: no-repeat;
+    @include sizeStar;
+  }
+
+  &__rating-star--not-active {
+    background-image: url("../images/star.svg");
+  }
+
+  &__feadback-text {
+    @include sizeFont;
+  }
+
+  &__price {
+    @include spaceBetween;
+    align-items: center;
+    @include sizeFont;
+    margin-top: 18px;
```

```suggestion
    @include spaceBetween;
    @include sizeFont;
    align-items: center;
    margin-top: 18px;
```
write all mixins on the top